src/EnergyPlus/MixedAir.cc

```diff
@@ -3924,6 +3906,15 @@ void VentilationMechanicalProps::CalcMechVentController(
 
                                         // Calculate zone maximum target CO2 concentration in PPM
                                         if (this->SystemOAMethod == DataSizing::SysOAMethod::ProportionalControlDesOcc) {
+                                            // Accumulate CO2 generation from people at design occupancy and current activity level
+                                            Real64 CO2PeopleGeneration = 0.0;
+                                            for (int PeopleNum = 1; PeopleNum <= state.dataHeatBal->TotPeople; ++PeopleNum) {
+                                                if (state.dataHeatBal->People(PeopleNum).ZonePtr != ZoneNum) continue;
```

Comment:
There are no per-zone PeopleNum lists I guess.

Reply:
Will address this with #9969